go/executor/pai.go

```diff
@@ -476,7 +476,7 @@ func executeOptimizeUsingOptFlow(pythonExecutor *pythonExecutor, stmt *ir.Optimi
 		dropTmpTables([]string{table}, pythonExecutor.Session.DbConnStr)
 	}
 
-	if len(stmt.Variables) > 1 {
+	if len(stmt.Variables) > 2 {
```

Comment:
Should it be `>=2` ?

Reply:
No. It should be `len(stmt.Variables) > 2`. The code inside the `if` statement is to create a table with the concatenated variable column. OptFlow has supported `len(stmt.Variables) == 2`, and we do not need to concatenate the variable column.

Reply:
I see.